README.md

```diff
@@ -51,9 +52,6 @@ See also dockerhub tags page: https://hub.docker.com/r/fluent/fluentd-kubernetes
 - `Graylog`
   - `docker pull fluent/fluentd-kubernetes-daemonset:v1.13.1-debian-graylog-1.0`
   - `docker pull fluent/fluentd-kubernetes-daemonset:v1.13-debian-graylog-1`
-- `Papertrail`
-  - `docker pull fluent/fluentd-kubernetes-daemonset:v1.13.1-debian-papertrail-1.0`
-  - `docker pull fluent/fluentd-kubernetes-daemonset:v1.13-debian-papertrail-1`
 - `Logzio`
```

Comment:
s3 is not multiarched. Only for amd64:
https://hub.docker.com/layers/fluent/fluentd-kubernetes-daemonset/v1.13.1-debian-logzio-amd64-1.0/images/sha256-49e70c93e717d18b83808c5a3cd20cd11871514a9930cf66f76aa87cdbfb3b6f?context=repo

Reply:
fixed